namesys/publisher.go

```diff
@@ -105,6 +106,11 @@ func ipnsEntryDataForSig(e *pb.IpnsEntry) []byte {
 		[]byte{})
 }
 
+var IpnsRecordValidator = &record.ValidChecker{
+	Func: ValidateIpnsRecord,
+	Sign: true,
```

Comment:
Also, getting rid of that field makes it really hard to tell which fields we have to sign (when creating records)